forge-gui/res/cardsfolder/upcoming/errant_and_giada.txt

```diff
@@ -0,0 +1,10 @@
+Name:Errant and Giada
+ManaCost:1 W U
+Types:Legendary Creature Human Angel
+PT:2/3
+K:Flash
+K:Flying
+S:Mode$ Continuous | Affected$ Card.TopLibrary+YouCtrl | AffectedZone$ Library | MayLookAt$ You | Description$ You may look at the top card of your library any time.
+S:Mode$ Continuous | Affected$ Card.TopLibrary+YouCtrl+withFlying,Card.TopLibrary+YouCtrl+hasKeywordFlash | MayPlay$ True | AffectedZone$ Library | Description$ You may cast spells with flash or flying from the top of your library.
+SVar:NonStackingEffect:True
```

Comment:
add some DeckHints for the stuff this card likes

Reply:
Added DeckHints